django/library/models.py

```diff
@@ -214,7 +234,7 @@ def to_codemeta(self):
         if self.orcid_url:
             codemeta["@id"] = self.orcid_url
         if self.affiliations.exists():
-            codemeta["affiliation"] = self.formatted_affiliations
+            codemeta["affiliation"] = self.formatted_json_affiliations
```

Comment:
this should be a rich object [schema.org/Organization](https://schema.org/Organization), maybe something like:

```
affiliation = self.json_affiliations[0] # assuming it exists
{
"@type": "Organization",
"@id": affiliation.ror_id,
"identifier": affiliation.ror_id,
"name": affiliation.name,
"url": affiliation.url,
"sameAs": affiliation.ror_id
}
```